### DISCUSSION TITLE
Nemo - pre-select "cancel" instead of "delete"

### DISCUSSION THREAD

**Discussion by HT-7** [linuxmint/discussions#137] [Ideas]
In user interface design, a destructive operation should never be pre-selected, so the pre-selected option on the "Are you sure you want to permanently delete …" dialogue box needs to be "cancel" instead of "delete".

By the way, the same applies to the file saving dialogue ("save as"). It pre-selects "replace". Either "cancel" needs to be pre-selected, or a third option "rename" needs to be added and that is pre-selected.

**Comment by Secret-chest** [comment]
I don't support just because I press space when one of these shows up.

**Reply by HT-7** [reply]
It could cause accidental deletion.

**Comment by Secret-chest** [comment]
> It could cause accidental deletion.

Should be an option then.
